modules/standard/Regex.chpl

```diff
@@ -966,6 +966,9 @@ record regex {
 
```

Comment:
Below: "Enumerates matches in the text as well as capture groups." --> something like "Yields matches and capture groups in `text`, continuing until ....."